### PR TITLE
#657: Add support for  INCRBY command

### PR DESCRIPTION
**Pull request by iamskp11**
### **Issue Fixed**
#657 

### PR Description
This PR adds support for INCRBY command, and have extensive integration tests for command.

### Test Results

Integration Tests
Pasted Graphic 9

### CODE REVIEW

**Comment by iamskp11** [comment]
Hey @JyotinderSingh @lucifercr07 @jujiale , requesting for your review on this small PR. Thanks!

File: integration_tests/commands/incr_test.go [hunk 1]
```diff
@@ -161,4 +161,144 @@ func TestINCR(t *testing.T) {
...
+			},
+		},
+	}
+
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+
+			FireCommand(conn, "DEL key")
+			FireCommand(conn, "DEL unsetKey")
+			FireCommand(conn, "DEL stringkey")
```
Comment (JyotinderSingh) [inline integration_tests/commands/incr_test.go:276]: Add defer calls before each of them so that the cleanup happens after the test runs

**Review by JyotinderSingh — CHANGES_REQUESTED** [review]
Thanks for adding support for incrby command! The changes book good. I've left a minor comment.

File: integration_tests/commands/incr_test.go (hunk 1 above)
Comment (iamskp11) [reply integration_tests/commands/incr_test.go]: @JyotinderSingh , Made changes in this [commit](https://github.com/DiceDB/dice/pull/676/commits/b678e9ef2568b340652d9dc0acdd7bbb44ecaf24). Also clubbed all keys deletion to single query.

**Review by JyotinderSingh — APPROVED** [review]
Thanks for adding support for `INCRBY` command @iamskp11! The changes look good and well-tested. Approved.

**Pull request merged**